### ISSUE TITLE
Bug: problem response arrays failing to parse

### ISSUE THREAD

**Issue by SoryRawyer** [openedx/tutor-contrib-aspects#511]
Responses to multiple-choice problems are stored as stringified arrays in the xAPI events. When the choices have single quotes in them, the response is quoted with double quotes, which breaks ClickHouse's ability to parse that field as an array of strings. However, escaping single quotes will incorrectly escape single-quoted sings in the response array. As an example, a multiple-choice response from the demonstration course appears in the xAPI events as `['a tree']` but appears in `problem_events` as `[\' a tree\']` when it should be unaltered from the xAPI form.

The `problem_events` materialized view should be updated to correctly handle double-quoted elements in the `responses` xAPI field.

**Comment by bmtcril** [comment]
Based on some research into how we've parsed these historically I think there is more work to be done that this, and it should be done in the event conversion since it's alllllll Python. So I'm going to close this one in favor of a new issue in ERB. We will have to change our parsing here again once that lands, but I won't know how to ticket that until we get the new values in.
